### PR TITLE
iCubTwente01 update

### PR DESCRIPTION
**Pull request by davidetome**
- See commit 017099a4634810a2dc4e0630714b54ca1bdc66dd
- See https://github.com/icub-tech-iit/fix/issues/260

### CODE REVIEW

File: iCubTwente01/hardware/motorControl/torso-eb5-j0_2-mc.xml [hunk 1]
```diff
@@ -1,90 +1,118 @@
...
-        <param name="motorNominalCurrents">     5000          5000          5000        </param>
-        <param name="motorPeakCurrents">        6000          6000          6000        </param>
-        <param name="motorOverloadCurrents">   15000         15000         15000        </param>
-        <param name="jntVelMax">                 1000          1000          1000        </param>
-        <param name="motorPwmLimit">            10000         10000         10000       </param>   
+        <param name="jntPosMax">                50                  30                  70                  </param>
+        <param name="jntPosMin">                -50                 -30                 -20                 </param>
+        <param name="jntVelMax">                1000                1000                1000                </param>
+        <param name="motorNominalCurrents">     4000                4000                4000                </param>
+        <param name="motorPeakCurrents">        5000                5000                5000                </param>
```
Comment (Nicogene) [inline iCubTwente01/hardware/motorControl/torso-eb5-j0_2-mc.xml]: These 2 values have been changed

File: iCubTwente01/hardware/motorControl/torso-eb5-j0_2-mc.xml [hunk 2]
```diff
@@ -1,90 +1,118 @@
...
+        <param name="fbkControlUnits">        metric_units          </param>
+        <param name="outputControlUnits">     machine_units         </param>
+        <param name="kp">           -711.11     -1066.66    -1066.66    </param>
+        <param name="kd">           0.00        0.00        0.00        </param>
+        <param name="ki">           -7111.09    -10666.64   -14222.18   </param>
+        <param name="maxOutput">    8000        8000        16000       </param>
+        <param name="maxInt">       750         1000        2000        </param>
+        <param name="stictionUp">   0           0           0           </param>
+        <param name="stictionDown"> 0           0           0           </param>
+        <param name="kff">          0           0           0           </param>
```
Comment (Nicogene) [inline iCubTwente01/hardware/motorControl/torso-eb5-j0_2-mc.xml]: The signs of the pids are changed :warning:

Comment (davidetome) [reply iCubTwente01/hardware/motorControl/torso-eb5-j0_2-mc.xml]: fixed

File: iCubTwente01/hardware/motorControl/torso-eb5-j0_2-mc.xml (hunk 1 above)
Comment (davidetome) [reply iCubTwente01/hardware/motorControl/torso-eb5-j0_2-mc.xml]: fixed

**Pull request merged**